source/content/guides/composer-convert.md

```diff
@@ -169,19 +214,31 @@ If you are using an exported config, you will need to move the configuration fil
 
 Locate the configuration files in your existing site and move them here. If they are stored in the files directory on your existing site, retrieve them via [SFTP](/sftp), as the Git clone would not contain them. The example project is configured to use this location.
 
-## Update to Latest Drupal Core
+## Set up a Multidev (Optional)
 
-The Pantheon Drupal 8 Composer repository may not depend on the latest secure release of Drupal Core. To keep your site safe, it's a good idea to pull the latest version in now -- before moving on -- by running:
+If your Pantheon account has access to [Multidev](/multidev), create a Multidev to push your new code to:
 
 ```bash{promptUser:user}
-composer update drupal/core --with-dependencies
+git push origin composerify && terminus env:create $site.dev composerify
```

Comment:
Here's the point where the intrepid adventurer may receive the error:

> Changing database version after environment creation is not currently supported.

If this happens, then remove the `database:` section from pantheon.yml. If the user does this, then they should also downgrade to Drupal 8 and upgrade to Drupal 9 later, after the database upgrade is available.

Reply:
It would also probably be better to have the user create the 'composerify' multidev before pushing the branch. If you do it the other way, Pantheon won't necessarily pick up changes to the composer.json file.